content/en/docs/admin/admission-controllers.md

```diff
@@ -563,15 +564,14 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
 
 ### SecurityContextDeny
 
-This admission controller will deny any pod that attempts to set certain escalating [SecurityContext](/docs/tasks/configure-pod-container/security-context/) fields. This should be enabled if a cluster doesn't utilize [pod security policies](/docs/concepts/policy/pod-security-policy/) to restrict the set of values a security context can take.
+This admission controller will deny any pod that attempts to set certain escalating [SecurityContext](/docs/user-guide/security-context) fields. This should be enabled if a cluster doesn't utilize [pod security policies](/docs/user-guide/pod-security-policy) to restrict the set of values a security context can take.
 
 ### ServiceAccount
 
-This admission controller implements automation for [serviceAccounts](/docs/tasks/configure-pod-container/configure-service-account/).
+This admission controller implements automation for [serviceAccounts](/docs/user-guide/service-accounts).
 We strongly recommend using this admission controller if you intend to make use of Kubernetes `ServiceAccount` objects.
 
-### Storage Object in Use Protection (beta)
-{{< feature-state for_k8s_version="v1.10" state="beta" >}}
+### Storage Object in Use Protection
 The `StorageObjectInUseProtection` plugin adds the `kubernetes.io/pvc-protection` or `kubernetes.io/pv-protection` finalizers to newly created Persistent Volume Claims (PVCs) or Persistent Volumes (PV). In case a user deletes a PVC or PV the PVC or PV is not removed until the finalizer is removed from the PVC or PV by PVC or PV Protection Controller. Refer to the [Storage Object in Use Protection](/docs/concepts/storage/persistent-volumes/#storage-object-in-use-protection) for more detailed information.
 
 ### ValidatingAdmissionWebhook (alpha in 1.8; beta in 1.9)
```

Comment:
This comment is really about lines 610++ below, but I can't add a comment there. All of that content relates to versions of Kubernetes that are no longer supported when 1.11 is released (only 1.7+ will be supported), so should they just be removed in the `release-1.11` branch (aka this commit)?